### ISSUE TITLE
Suggestion and Help request Bonus

### ISSUE THREAD

**Issue by farissalih87** [jekkos/opensourcepos#487]
I am using ospos in our store and we have bonus system , like if you buy 10 I have to give you 2 more as a bonus I want to include that at the sales form, receipts, invoices and reports and both the main quantity and the bonus shuld be  deducted from the item stok  
Need some help with my poor programming skills.

**Comment by jekkos** [comment]
Think the title for these issues are sometimes so vague that we lose overview pretty fast. Maybe we should consider doing a template for feature requests as well.

**Comment by freemail624** [comment]
I think i found something like this [Buy 2, get 1 free](http://haseydesign.com/flexi-cart/standard_library/insert_discount_item_to_cart/304)
 [Features of flexi cart](http://haseydesign.com/flexi-cart/lite_library/features)
[Item Discount Examples](http://haseydesign.com/flexi-cart/lite_library/item_discount_examples)

**Comment by daN4cat** [comment]
Yes, sales register (or cart) should have many more options. At the moment there is just overall fix discount and customer based discount (I added it last week).

Other options would be:
  - fixed amount discount by customer
  - special offers like buy 3 pay 2 (or 2 pay 1)
  - buy 10 times get 1 free (e.g. point cards)
  - accumulate points and spend them (e.g. loyalty cards)

I guess there are many more.

All nice, but it's work that needs somebody to take ownership and contribute or we get somebody to do pay them.

Personally besides the customer discount I don't have need for such additional features. Maybe one day when most of the features are completed.
